modules/doc/content/application_development/build_system.md

```diff
@@ -185,92 +210,112 @@ You'll probably want to add the path above to your PATH.
+!style! halign=left
+MOOSE's build system has the ability to create installed binaries suitable for use with the conda
+package manager, or installation on a shared computing resource such as Sawtooth. The `install`
+target will copy the binary, and required libraries to a file tree based on the prefix you
+configured with (set by the variable `PREFIX` or the `--prefix` argument).
+!style-end!
 
 ```
-$ cd moose
-$ ./configure --prefix=<installation path>
-$ cd <application_dir>
-$ make [make options]
-$ make install
+cd moose
+./configure --prefix=<installation path>
+cd <application_dir>
+make [make options]
+make install
 ```
 
-Additionally each application may wish to make an one or more file structures available for end users to install into their own environment. This allows for end-users to install and run the application test suite, examples, tutorials, or other inputs of interest. To customize what is available for installation, the application developer will define the list of source directories that can be installed by end users in their application Makefile. For example:
+Additionally each application may wish to make an one or more file structures available for end
+users to install into their own environment. This allows for end-users to install and run the
+application test suite, examples, tutorials, or other inputs of interest. To customize what is
+available for installation, the application developer will define the list of source directories
+that can be installed by end users in their application Makefile. For example:
 
 ```
 INSTALLABLE_DIRS := tests examples tutorials
 ```
 
+!style! style=position:relative;top:-15px;left:5px;font-style:italic;font-size:small;
 These paths should all be relative to the root of the application.
+!style-end!
+
 
-To give application developers the ability to customize the layout of the installed directories it's possible to specify the destination path as well by using a "key/value" syntax. For example:
+To give application developers the ability to customize the layout of the installed directories it's
+possible to specify the destination path as well by using a "key/value" syntax. For example:
 
 ```
 INSTALLABLE_DIRS := test/tests->tests examples/01->example_01 examples/02->example_02 tutorials
 ```
 
-In this case, if the user chooses to install each of these directories into a writable workspace the resulting directory structure will look like this:
+In this case, if the user chooses to install each of these directories into a writable workspace the
+resulting directory structure will look like this:
 
-```
+```bash
 $ ls -l
 tests  example_01  example_02  tutorials
 ```
 
 !alert warning
```

Comment:
```suggestion
!alert! warning
```
Since we are using two lines after this alert, let's use the full syntax to ensure it is rendered properly